Documentation/compatibility.md

```diff
@@ -27,75 +27,9 @@ v0.26.1
 
 ## Prometheus
 
-The versions of Prometheus compatible with the Prometheus Operator are:
+Prometheus Operator supports Prometheus from the before last [Prometheus TLS release](https://prometheus.io/docs/introduction/release-cycle/) up to the latest stable. The versions of Prometheus currently compatible with the Prometheus Operator are:
```

Comment:
the wording is a bit more "aggressive" than what I had in mind. IMHO we don't need to drop support for versions before LTS n-1. But it should be clear that the testing only covers TLS n-1 -> latest.
What about this?

```suggestion
Prometheus Operator supports all Prometheus versions >= v2.0.0. The operator's end-to-end tests verify that the operator can deploy the following Prometheus versoins:
```